lib/modules/onc_program/test/bulk_group_validation_sequence_test.rb

```diff
@@ -606,6 +606,14 @@
 
       assert pass_exception.message == 'Successfully validated 0 resource(s).'
     end
+
+    it 'passes when no Medication export' do
+      pass_exception = assert_raises(Inferno::PassException) do
```

Comment:
This should be an omit, as my suggestions are above, so we check an OmitException if you accept that change request?

Reply:
fixed